src/main/java/org/tron/core/capsule/BlockCapsule.java

```diff
@@ -302,29 +302,6 @@ public long getTimeStamp() {
 
   private StringBuffer toStringBuff = new StringBuffer();

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". ([squid:S1149](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1149))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0NlHK86Hco_1qj&open=AWalU-0NlHK86Hco_1qj)

[ISSUE_KEY]: AWalU-0NlHK86Hco_1qj